### ISSUE TITLE
Atom-Typescript error: Cannot read property 'NaN' of undefined

### ISSUE THREAD

**Issue by ZuBB** [TypeStrong/atom-typescript#1474]
Hi! 

Just got this issue. I am sorry I do not have strong/reliable steps to reproduce this issue. I can just describe what happened:
 * I had some ts files opened in atom;
 * i moved them in console (outside of atom)
 * back to editor and got some ts errors (red underscore line)
 * I know sometime those errors/warnings are false, so I started to close all opened files and after some closed file (or all of them; I made closing very fast) I got this error

**Atom**: 1.32.2 x64
**Electron**: 2.0.9
**OS**: Mac OS X 10.13.6
**Thrown From**: [atom-typescript](https://github.com/TypeStrong/atom-typescript) package 12.7.0

### Stack Trace

Atom-Typescript error: Cannot read property 'NaN' of undefined

```
At TypeError: Cannot read property 'NaN' of undefined

TypeError: Cannot read property 'NaN' of undefined
    at TextEditorComponent.renderedScreenLineForRow (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:80730)
    at TextEditorComponent.screenPositionForPixelPosition (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:106937)
    at TextEditorComponent.screenPositionForMouseEvent (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:98290)
    at Object.bufferPositionFromMouseEvent (/packages/atom-typescript/dist/main/atom/tooltips/util.js:8:10)
    at TooltipController.initialize (/packages/atom-typescript/dist/main/atom/tooltips/controller.js:20:33)
    at /packages/atom-typescript/dist/main/atom/tooltips/controller.js:10:36)
    at TooltipManager.showExpressionType (/packages/atom-typescript/dist/main/atom/tooltips/manager.js:87:31)
    at exprTypeTimeout.window.setTimeout (/packages/atom-typescript/dist/main/atom/tooltips/manager.js:47:69)
```

### Commands

```
     -5:26.3.0 vim-mode-plus:move-to-beginning-of-line (input.hidden-input)
  4x -5:26 vim-mode-plus:move-to-next-word (input.hidden-input)
  5x -5:23.4.0 vim-mode-plus:move-down (input.hidden-input)
  3x -5:21.6.0 vim-mode-plus:move-up (input.hidden-input)
  4x -5:19.5.0 vim-mode-plus:move-down (input.hidden-input)
     -4:48.6.0 core:save (input.hidden-input)
     -4:48.3.0 core:close (input.hidden-input)
     -4:47 core:save (input.hidden-input)
     -1:44.9.0 tree-view:move (div.tool-panel.tree-view)
     -1:44.3.0 editor:move-to-first-character-of-line (input.hidden-input)
     -1:43.5.0 editor:move-to-end-of-screen-line (input.hidden-input)
  3x -1:43.2.0 editor:move-to-beginning-of-word (input.hidden-input)
     -1:39.8.0 editor:move-to-end-of-line (input.hidden-input)
     -1:38.1.0 core:confirm (input.hidden-input)
     -1:31.8.0 tree-view:copy-project-path (span.name.icon.icon-file-directory)
  4x -0:09.6.0 core:close (div.tool-panel.tree-view)
```

### Non-Core Packages

```
angular-2-typeScript-snippets 0.7.0 
atom-beautify 0.33.4 
atom-clock 0.1.16 
atom-ide-ui 0.13.0 
atom-live-server 2.3.0 
atom-typescript 12.7.0 
busy-signal 1.4.3 
compare-files 0.8.1 
editor-stats 0.17.0 
editorconfig 2.2.2 
emmet 2.4.3 
ex-mode 0.18.0 
git-control 0.9.0 
git-plus 8.1.0 
go-plus 5.9.1 
goto-definition 1.3.4 
ide-typescript 0.7.9 
intentions 1.1.5 
language-scala 1.1.9 
linter 2.2.0 
linter-ui-default 1.7.1 
minimap 4.29.9 
open-recent 5.0.0 
pretty-json 2.0.3 
pull-request 0.2.1 
show-invisibles-plus 0.3.1 
svg-preview 0.14.0 
sync-settings 0.8.6 
todo-show 2.3.2 
typescript-modules-helper 0.8.0 
vim-mode-plus 1.36.0 
```

**Comment by lierdakil** [comment]
Thanks for reporting.

This seems to basically be a weird synchronisation issue: a text editor was destroyed, but a tooltip was pending on it, so when time came to show the tooltip, there wasn't an editor already, which in turn produced this error. I'm pretty sure chances of this happening are rather slim, and this doesn't interfere with anything (aside from throwing out a scary error window), so I think I'm going to ignore this for now.

Let me know if you run into this again, perhaps I'm underestimating the probability of this happening. Also, anyone else running into this, feel free to :+1: the OP, this will help us gauge the incidence of the issue. Thanks.

**Comment by ZuBB** [comment]
yes, sure

@lierdakil thanks for a detailed reply

**Comment by lenovouser** [comment]
I just got this too, I don't exactly know how I did it though 😕

**Comment by senyaak** [comment]
+1, tried to save - but file didn't response, so I closed it and got this error.

**Comment by lierdakil** [comment]
Should be fixed in v12.7.3. Please open new issues if you run into something similar.
